solidity/contracts/converter/BancorConverterUpgrader.sol

```diff
@@ -195,11 +199,21 @@ contract BancorConverterUpgrader is IBancorConverterUpgrader, ContractRegistryCl
             address connectorAddress = _oldConverter.connectorTokens(i);
             (virtualBalance, weight, isVirtualBalanceEnabled, , ) = _oldConverter.connectors(connectorAddress);
 
-            IERC20Token connectorToken = IERC20Token(connectorAddress);
-            _newConverter.addConnector(connectorToken, weight, isVirtualBalanceEnabled);
-
-            if (isVirtualBalanceEnabled)
-                _newConverter.updateConnector(connectorToken, weight, isVirtualBalanceEnabled, virtualBalance);
+            // Ether reserve
+            if (connectorAddress == address(0)) {
+                _newConverter.addETHReserve(weight);
+            }
+            // Ether reserve token
+            else if (connectorAddress == address(etherToken)) {

```

Comment:
can just handle ETH reserve and EtherToken reserve in the same block here - exact same implementation

Reply:
@yudilevi : Correct, but due to your previous comment on this section, as well as due to the `if`/`else` structure of the other function (`transferConnectorsBalances`), I've decided to write it this way for better clarity. This should yield very little impact on deployment gas-cost, and no impact at all on runtime gas-cost.

Reply:
maybe reconsider after the `else` block is fixed to also cover ETH reserve

Reply:
@yudilevi : Sorry, I don't get it.

Reply:
Nm, I think it's fine - the 2nd case is temporary anyway